src/server/server.py

```diff
@@ -236,7 +236,7 @@ async def start_client(
             self.log.info(f"Connecting: {url}, Peer info: {target_node}")
             try:
                 ws = await session.ws_connect(
-                    url, autoclose=False, autoping=True, heartbeat=300, ssl=ssl_context, max_msg_size=50 * 1024 * 1024
+                    url, autoclose=False, autoping=True, heartbeat=30, ssl=ssl_context, max_msg_size=50 * 1024 * 1024
```

Comment:
I'm still not sure I get the logic that this will help keep harvester connected. My bet is we're just going to get more strange disconnects globally.